### PR TITLE
Add driver override to steering module

### PR DESCRIPTION
**Pull request by LucasBuckland**
Prior to this commit we didn't have a good way of disabling on driver override on steering wheel. This commit adds the functionality to detect a driver override on steering wheel, and then disable properly.

### CODE REVIEW

File: firmware/steering/kia_soul_ps/steering_control_module.ino [hunk 1]
```diff
@@ -350,6 +350,74 @@ void disable_control( )
...
+
+    filtered_torque_a = 
+        ( torque_filter_alpha * torque_sensor_a ) +
+            ( ( 1.0 - torque_filter_alpha ) * filtered_torque_a );
+
+    filtered_torque_b = 
+        ( torque_filter_alpha * torque_sensor_b ) +
+            ( ( 1.0 - torque_filter_alpha ) * filtered_torque_b );
+
+    Serial.print( filtered_torque_a );
```
Comment (ltbj) [inline firmware/steering/kia_soul_ps/steering_control_module.ino]: We should remove these, or alternatively make them DEBUG_PRINT statements.

**Comment by ltbj** [comment]
Please hold off on merging this until we've tested on the road.

File: firmware/steering/kia_soul_ps/steering_control_module.ino (hunk 1 above)
Comment (LucasBuckland) [inline firmware/steering/kia_soul_ps/steering_control_module.ino]: Oversight. Will remove.

**Comment by LucasBuckland** [comment]
This has been tested in a wide range of possible speeds and steering angles and works well.

**Pull request merged**